src/components/GameItem.js

```diff
@@ -9,17 +9,39 @@ class GameItem extends Component {
     layer: PropTypes.number.isRequired,
   }
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      spottedItem: false	      
+    };
+  }	  
+
+   spottedItem = () => {
+    this.setState({
+      spottedItem: !this.state.spottedItem
```

Comment:
Small bug: This ends up toggling the item! I can click on one piece of litter a bunch and get a bunch of points ;-)